.github/workflows/integration_test.yaml

```diff
@@ -238,7 +238,12 @@ jobs:
       - uses: canonical/setup-lxd@v0.1.1
       - name: Extract charm name
         working-directory: ${{ inputs.working-directory }}
-        run: echo "CHARM_NAME=$([ -f metadata.yaml ] && yq '.name' metadata.yaml || echo UNKNOWN)" >> $GITHUB_ENV
+        run: |
+          CHARM_NAME="$([ -f metadata.yaml ] && yq '.name' metadata.yaml || echo UNKNOWN)"
+          if [ "$CHARM_NAME" == "UNKNOWN" ]; then
+            CHARM_NAME="$([ -f charmcraft.yaml ] && yq '.name' charmcraft.yaml || echo UNKNOWN)"
+          fi
```

Comment:
`yq` supports looking into multiple files and it works if file doesn't exists, so I think that something like:
`yq .name metadata.yaml charmcraft.yaml 2>/dev/null | grep -v null || echo UNKNOWN` should work

Reply:
I'd rather not disregard error output, it any issue arise, it will make it harder to spot. Also, what you propose doesn't seem much shorter

Reply:
Yeap indeed, that works but less explicit